rangeproof/splitutils.go

```diff
@@ -8,13 +8,13 @@ import (
 )
 
 type (
-	// SquareSplitter provides a combined interface for all facets describing a method for spliting positive numbers into a sum of squares.
+	// SquareSplitter provides a combined interface for all facets describing a method for splitting positive numbers into a sum of squares.
 	SquareSplitter interface {
-		// Number of bits per square
+		// Ld - Number of bits per square
 		Ld() uint
-		// Number of squares in result
+		// SquareCount - Number of squares in result
 		SquareCount() int
-		// Actual splitting function, on input delta, should return array x such that sum_i x_i^2 = delta and len(x) = SquareCount()
+		// Split - Actual splitting function, on input delta, should return array x such that sum_i x_i^2 = delta and len(x) = SquareCount()
```

Comment:
These seem to be the only places where you use a dash as opposed to a full sentence. For consistency I would prefer a full sentence.